packages/components/src/internal/components/samples/utils.tsx

```diff
@@ -83,11 +83,21 @@ export function getSampleStatusType(row: any): SampleStateType {
 }
 
 export function getSampleStatus(row: any): SampleStatus {
+    let label;
+    // Issue 45269. If the state columns are present, don't look at a column named 'Label'
+    let field = caseInsensitive(row, SAMPLE_STATE_COLUMN_NAME)
+    if (field) {
+        label = field.displayValue;
+    } else {
+        field = caseInsensitive(row, 'SampleID/' + SAMPLE_STATE_COLUMN_NAME);
+        if (field) {
+            label = field.displayValue;
+        } else {
+            label = caseInsensitive(row, 'Label')?.value;
+        }
+    }
     return {
-        label:
-            caseInsensitive(row, SAMPLE_STATE_COLUMN_NAME)?.displayValue ||
-            caseInsensitive(row, 'SampleID/' + SAMPLE_STATE_COLUMN_NAME)?.displayValue ||
-            caseInsensitive(row, 'Label')?.value,
+        label,
```

Comment:
I am not seeing what the change is here...

Reply:
If your same does not have a status, then the first two checks for a display value will return `undefined` in the original code, which means it will pick up the value of a field called `Label` if you have one, but that likely has nothing to do with the sample status.  So, if there is a field called `SampleState`, we want to use its value, even if it is `undefined`. 

The use of `Label` here is specifically for the case of display data from the dataStates table itself, where `Label` is the label for he status and the other two fields won't exist.

Reply:
Ah yes, got it. I see the change now. Thanks.